docs/data/sources/google-tag-manager-client-legacy.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Amplitude Analytics Legacy
+description: Official Google Tag Manager template for Amplitude Analytics for data collection.
+---
+
+Client-side Google Tag Manager Template for Amplitude Analytics. The tag utilizes the [Amplitude JavaScript SDK](../../data/sdks/javascript/index/) for data collection.
+
+!!!deprecated Legacy
+    This is a legacy GTM template. For the most up-to-date implementation. It's recommended to use the [latest template](./google-tag-manager-client.md) for optimal implementation.
+
+!!!info Resources
+    [:simple-googletagmanager: GTM Template Gallery](https://tagmanager.google.com/gallery/#/owners/amplitude/templates/amplitude-gtm-template) · [:material-github: GitHub](https://github.com/amplitude/amplitude-gtm-template)
+
+## Tutorial video
```

Comment:
```suggestion
This video tutorial walks through the implementation basics. 
```

Unless you're including more sections in this doc, we don't need a heading here.